### PR TITLE
Merge all tools' apps into Kitkit Launcher

### PR DESCRIPTION
**Pull request by agarg8487**
|PR checklist|Status|
|-----|:------:|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|Yes|
|Is code analysis task executed and issues resolved in the CPP/native code?|NA|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|No|
|Is code commenting done properly for the changes made?|Yes|
|Is correct variable naming convention followed?|Yes|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|Yes|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|Yes|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|NA|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|Yes|

### CODE REVIEW

**Review by ankitk20 — DISMISSED** [review]

File: launcher/app/src/main/java/com/maq/xprize/blackboard/activity/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    }
+
+    @Override
+    public void onWindowFocusChanged(boolean hasFocus) {
+        if (hasFocus == true) {
+            mDecorView.setSystemUiVisibility(mUiOption);
+        }
+    }
+
+    ////////////////////////////////////////////////////////////////////////////////
```
Comment (harry6396) [inline launcher/app/src/main/java/com/maq/xprize/blackboard/activity/MainActivity.java]: Remove this

File: launcher/app/src/main/java/com/maq/xprize/blackboard/utility/Log.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            int prev_line = 0;
+
+            String class_name = getClassName(a[2].getClassName());
+
+            if (a.length > 2) {
+                prev_method_name = a[3].getMethodName();
+                prev_line = a[3].getLineNumber();
+            }
+
+            // if (prev_class_name.length() > 0 && prev_class_name.equalsIgnoreCase(class_name) == false)
```
Comment (harry6396) [inline launcher/app/src/main/java/com/maq/xprize/blackboard/utility/Log.java]: Remove this

File: launcher/app/src/main/java/com/maq/xprize/blackboard/utility/Log.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+
+    private static String getClassName(String val) {
+        String rtn = val;
+
+        try {
+            if (rtn.contains(".")) {
+                rtn = rtn.substring(rtn.lastIndexOf(".") + 1);
+            } else {
+                // class_name = class_name.replace(context.getPackageName() + ".", "");
```
Comment (harry6396) [inline launcher/app/src/main/java/com/maq/xprize/blackboard/utility/Log.java]: Remove this

File: launcher/app/src/main/java/com/maq/xprize/drawingcoloring/activity/DrawingColoringActivity.java [hunk 4]
```diff
@@ -0,0 +1,709 @@
...
+                                startChalkingSound();
+
+                            }
+                        }
+                    } else {
+                        stopChalkingSound();
+                    }
+                }
+
+//                if (mbSoundingChalk == true) {
```
Comment (harry6396) [inline launcher/app/src/main/java/com/maq/xprize/drawingcoloring/activity/DrawingColoringActivity.java]: Remove this

File: launcher/app/src/main/java/com/maq/xprize/drawingcoloring/utility/Log.java [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+
+    private static String getClassName(String val) {
+        String rtn = val;
+
+        try {
+            if (rtn.contains(".")) {
+                rtn = rtn.substring(rtn.lastIndexOf(".") + 1);
+            } else {
+                // class_name = class_name.replace(context.getPackageName() + ".", "");
```
Comment (harry6396) [inline launcher/app/src/main/java/com/maq/xprize/drawingcoloring/utility/Log.java]: Remove commented code

File: launcher/app/src/main/java/com/maq/xprize/kitkitlauncher/hindi/ToolsActivity.java [hunk 6]
```diff
@@ -96,18 +94,11 @@ public void onClick(View v) {
...
-                        i.setComponent(new ComponentName("com.maq.xprize.drum", "com.maq.xprize.drum.activity.MainActivity"));
-                        startActivity(i);
-                        KitKitLogger logger = ((LauncherApplication) getApplication()).getLogger();
-                        logger.logEvent("ToolsActivity", "start_drum", "", 0);
-                    } catch (Exception e) {
-                        redirectPopup(packageName, "Drum: किटकिट स्कूल - Kitkit School module");
-                    }
-
+//                    Calling Drum's main activity
+                    Intent intent = new Intent(ToolsActivity.this, DrumActivity.class);
```
Comment (harry6396) [inline launcher/app/src/main/java/com/maq/xprize/kitkitlauncher/hindi/ToolsActivity.java]: Use try and catch to handle the Exception with every intent

**Review by harry6396 — APPROVED** [review]

**Review by ankitk20 — APPROVED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**